GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -571,6 +567,30 @@ private void CheckNotInsideExistingRepo(string normalizedEnlistmentRootPath)
             }
 
             GitProcess.Result forceCheckoutResult = git.ForceCheckout(branch);
+            if (forceCheckoutResult.HasErrors && forceCheckoutResult.Errors.IndexOf("unable to read tree") > 0)

```

Comment:
@derrickstolee In reviewing this PR, it occurs to me that we could use our own `GitIndexGenerator` here instead of relying on `git checkout` while we're not yet mounted. Any arguments against that? (Not blocking this PR, but it would be a more robust fix and not language-dependent)

Reply:
I created #397 to track that idea.